lib/unexpectedHttp.js

```diff
@@ -4,6 +4,10 @@ var http = require('http'),
     messy = require('messy'),
     _ = require('underscore');
 
+function isStream(thing) {
+    return typeof thing.pipe === 'function';
```

Comment:
Yeah, was in two minds about that too - where it's used the null/undefined check was already done by the surrounding if, but it's not exactly a general test. I'm actually inclined just to move the `typeof <...>.pipe` bit inline wth a comment it's a check for a stream